packages/sdk/src/credential/teamsUserCredential.browser.ts

```diff
@@ -119,7 +119,7 @@ export class TeamsUserCredential implements TokenCredential {
 
             let resultJson: any = {};
             try {
-              resultJson = JSON.parse(result);
+              resultJson = typeof result == "string" ? JSON.parse(result) : result;
```

Comment:
if android only returns a token string, we may not able to login users (since we cannot set session storage correctly)?  @KennethBWSong

Reply:
I don't have time to debug the app now, but @formulahendry said it works on android after updating this line.

Reply:
sorry I understand the code incorrectly. The string should be a json string instead a pure token string.